include/secp256k1_silentpayments.h

```diff
@@ -53,7 +56,7 @@ typedef struct secp256k1_silentpayments_recipient {
  *  public keys are excluded from silent payments eligible inputs; see BIP352
  *  for more information.
  *
- *  `outpoint_smallest` refers to the smallest outpoint lexicographically
+ *  `outpoint_smallest36` refers to the smallest outpoint lexicographically
```

Comment:
in commit 32d8077cb7d288c7114dcf8d7b831d7ada590bd0: minimum-diff pedantic nit: this change, as well as all other single-line changes in the same commit below should already be part of an earlier commit